README.md

```diff
@@ -6,34 +6,40 @@ Sample applications for using the [OpenTok.js](https://tokbox.com/developer/sdks
 
 The code for this sample is found the following subdirectories:
 
-* Angular-Basic-Video-Chat ([demo](https://opentok.github.io/opentok-web-samples/Angular-Basic-Video-Chat/)) ([source](https://github.com/opentok/opentok-web-samples/tree/master/Angular-Basic-Video-Chat)) -- This sample shows you how to connect to an OpenTok session, publish a stream, and
+* Angular-Basic-Video-Chat ([demo](https://opentok.github.io/opentok-web-samples/Angular-Basic-Video-Chat/)) ([source](https://github.com/opentok/opentok-web-samples/tree/main/Angular-Basic-Video-Chat)) -- This sample shows you how to connect to an OpenTok session, publish a stream, and
   subscribe to a stream using [Angular 5](https://angular.io/).
 
-* Basic Video Chat ([demo](https://opentok.github.io/opentok-web-samples/Basic%20Video%20Chat/)) ([source](https://github.com/opentok/opentok-web-samples/tree/master/Basic%20Video%20Chat)) -- This sample shows you how to connect to an OpenTok session, publish a stream, and
+* Basic-Audio-Transformer ([demo](https://opentok.github.io/opentok-web-samples/Basic-Audio-Transformer/)) ([source](https://github.com/opentok/opentok-web-samples/tree/main/Basic-Audio-Transformer)) -- This sample application shows how to use a basic audio transformer with the Vonage video APIs. It is very similar to the Basic Video Chat example but it adds a low-pass filter to the published audio.
+
+* Basic-Background-Blur ([demo](https://opentok.github.io/opentok-web-samples/Basic-Background-Blur/)) ([source](https://github.com/opentok/opentok-web-samples/tree/main/Basic-Background-Blur)) -- This sample application shows how to add background blur with the Vonage video APIs and Vonage ML Transformers library.
```

Comment:
Should we follow the same pattern as the audio/video transformers in terms of the description?

Reply:
I thought I did ;-) 
What phrasing do you suggest?

Reply:
I thought something like `This sample application shows how to use filter with the Vonage video APIs. It is very similar to the Basic Video Chat example but it adds a background blur filter to the published audio.` It's super nit-picky though, I think it's fine as is.